src/views/Viewer.vue

```diff
@@ -595,7 +595,7 @@ export default {
 			fileInfo = this.fileList[this.currentIndex]
 
 			// show file
-			this.currentFile = new File(fileInfo, mime, handler.component)
+			this.currentFile = new File(fileInfo, mime, this.components[mime])
```

Comment:
If we use the handler instead of the registered aliases, we break all aliases on file opening.

Reply:
We could check again if the override handler is specified, if so we can use the original handler coponent?

```js
const component = overrideHandlerId ? handler.component : this.components[mime];
this.currentFile = new File(fileInfo, mime, component)
```

Reply:
No, we should be consistent on how we do things for other File creations :)
https://github.com/nextcloud/viewer/blob/bbcd4ef37c6ccc54f4f1dc6e19c95b69898b7ff0/src/views/Viewer.vue#L613
https://github.com/nextcloud/viewer/blob/bbcd4ef37c6ccc54f4f1dc6e19c95b69898b7ff0/src/views/Viewer.vue#L637
https://github.com/nextcloud/viewer/blob/bbcd4ef37c6ccc54f4f1dc6e19c95b69898b7ff0/src/views/Viewer.vue#L647